src/style.css

```diff
@@ -1 +1,68 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+* {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-weight: 300;
+  font-style: normal;
+  color: #3d4e61;
+  box-sizing: border-box;
+}
```

Comment:
You're using the `*` selector to set the font family, font weight, font style, color, and box-sizing for all elements. This is not recommended as it can impact performance. It's better to set these properties on specific elements that require them. (Refer to the CSS checklist)